### PR TITLE
Continue renaming things according to #16.

### PR DESCRIPTION
**Pull request by yakutovicha**

### CODE REVIEW

File: aiidalab_eln/cheminfo/exporter.py [hunk 1]
```diff
@@ -6,65 +6,63 @@
...
     source_info = {
-        "uuid": isotherm.uuid,
+        "uuid": node.uuid,
         "url": aiidalab_instance,
         "name": "Isotherm simulated using the isotherm app on AiiDAlab",
     }
     meta = {
-        "adsorptive": adsorptive,
-        "temperature": isotherm["temperature"],
+        "adsorptive": "N2",
```
[the remark this answers is not in the document]
Comment (yakutovicha) [reply aiidalab_eln/cheminfo/exporter.py]: done

File: aiidalab_eln/base_connector.py [hunk 2]
```diff
@@ -14,8 +14,6 @@ def __init__(self, **kwargs):
 
         Args:
             eln_instance (str): URL which points to the eln instance.
-            user (str): ELN user.
```
Comment (kjappelbaum) [inline aiidalab_eln/base_connector.py:17]: maybe username would have been a more explicit name, but not too important, maybe to mention in the docstring

File: aiidalab_eln/base_connector.py [hunk 3]
```diff
@@ -14,8 +14,6 @@ def __init__(self, **kwargs):
 
         Args:
             eln_instance (str): URL which points to the eln instance.
```
Comment (kjappelbaum) [inline aiidalab_eln/base_connector.py:16]:
```suggestion
            eln_instance (str): URL which points to the ELN instance.
```

File: aiidalab_eln/base_connector.py [hunk 4]
```diff
@@ -14,8 +14,6 @@ def __init__(self, **kwargs):
...
-            user (str): ELN user.
-            token (str): ELN access token (or user password).
             eln_type (str): ELN type, e.g. "cheminfo".
```
Comment (kjappelbaum) [inline aiidalab_eln/base_connector.py:17]:
```suggestion
            eln_type (str): ELN type, e.g. "cheminfo" or "openbis".
```

maybe we could even make this an `enum`

File: aiidalab_eln/cheminfo/__init__.py [hunk 5]
```diff
@@ -149,12 +149,11 @@ def export_data(self):
...
-                filename=self.file_name,
+                node=self.node,
+                file_name=self.file_name,
             )
         elif self.node.node_type == "data.cif.CifData.":
-            export_cif(sample, self.node, filename=self.file_name)
+            export_cif(sample, self.node, file_name=self.file_name)
 
     def import_data(self):
         """Import data object from Cheminfo ELN to AiiDAlab."""
```
Comment (kjappelbaum) [inline aiidalab_eln/cheminfo/__init__.py:159]:
```suggestion
        """Import data object from cheminfo ELN to AiiDAlab."""
```

File: aiidalab_eln/cheminfo/exporter.py [hunk 6]
```diff
@@ -1,70 +1,82 @@
 # -*- coding: utf-8 -*-
 """Export AiiDA objects to the Cheminfo ELN."""
```
Comment (kjappelbaum) [inline aiidalab_eln/cheminfo/exporter.py:2]:
```suggestion
"""Export AiiDA objects to the cheminfo ELN."""
```

File: aiidalab_eln/cheminfo/importer.py [hunk 7]
```diff
@@ -9,16 +9,16 @@
 def import_cif(sample, **kwargs):
     """Import CIF object from Cheminfo sample to AiiDAlab node."""
```
Comment (kjappelbaum) [inline aiidalab_eln/cheminfo/importer.py:10]:
```suggestion
    """Import CIF object from sample in cheminfo ELN to AiiDA node."""
```

File: aiidalab_eln/cheminfo/importer.py [hunk 8]
```diff
@@ -9,16 +9,16 @@
...
-    file = io.BytesIO(bytes(content, "utf8"))
-    node = object_type(file=file)
+    file_content = sample.get_data(data_type="xray", file_name=kwargs["file_name"])
+    file_object = io.BytesIO(bytes(file_content, "utf8"))
+    node = object_type(file=file_object)
     return node
 
 
 def import_pdb(sample, **kwargs):
     """Import PDB object from Cheminfo sample to AiiDAlab node."""
```
Comment (kjappelbaum) [inline aiidalab_eln/cheminfo/importer.py:19]:
```suggestion
    """Import PDB object from sample in cheminfo ELN to AiiDA node."""
```

**Review by kjappelbaum — APPROVED** [review]
many thanks, lgtm

**Pull request merged**